Assets/Scripts/Audio/AudioManager/MechanicAudioManager.cs

```diff
@@ -5,12 +5,15 @@ public class MechanicAudioManager : AbstractAudioManager {
 
     //Audioclips
     public AudioClip buttonClip;
+	public AudioClip unlockDoorClip;
```

Comment:
They are going to be accessed by the audio source itself, these variables are here so that we can just change the volume at one place